stock_prices/stock_prices.py

```diff
@@ -1,15 +1,24 @@
 #!/usr/bin/python
 
 import argparse
+import math
+
 
 def find_max_profit(prices):
-  pass
+    max_profit, min_price = -math.inf, math.inf
+    for price in prices:
```

Comment:
very efficient, compact solution. I think this is the fewest lines of code to passing the test that i've seen so far.